templates/make_manifest

```diff
@@ -32,7 +32,7 @@ if [[ $DIRECTOR_NAME = "Bosh Lite Director" ]]; then
 fi
 
 if [[ $infrastructure = "aws-ec2" ]]; then
-  if [[ $DIRECTOR_CPI != "aws" ]]; then
+  if [[ $DIRECTOR_CPI != "aws"* ]]; then
```

Comment:
This solve the problem with wrong Director CPI detection, actually AWS Director CPI name "aws_cpi".
```
bosh status | grep CPI
CPI aws_cpi
```
If we use "aws" instead of "aws"* the result is:
```
Not targeting an AWS BOSH. Please use 'bosh target' before running this script
```